airbyte/_connector_base.py

```diff
@@ -380,6 +383,14 @@ def _peek_airbyte_message(
                 )
             return
 
+        if (
+            message.type == "CONTROL"
+            and message.control.type == "CONNECTOR_CONFIG"
+            and self.config_change_callback is not None
+        ):
+            self.config_change_callback(message.control.config, message.control.emitted_at)
```

Comment:
WDYT of only passing the new config and omitting the emitted_at input? Or do you think the developer may need both?

Reply:
My original thought was that the developer may want to make sure the latest config was the most recent. But in theory they should always get called in order. I reverted to just sending the config.